front/components/actions/websearch/utils.ts

```diff
@@ -1,4 +1,4 @@
-import type { CitationType } from "@dust-tt/sparkle/dist/cjs/components/Citation";
+import type { CitationType } from "@dust-tt/sparkle/dist/esm/components/Citation";
```

Comment:
Seen IRL, we will find a way to improve the type situation afterward.

Reply:
We'll add a proper exports for types, removing all direct imports for now